scripts/robust.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$id": "http://fixme.com/robust.json",
+  "$schema": "http://json-schema.org/schema#",
+  "title" : "ROBUST bug file schema", 
+  "type": "object",
+  "properties": {
+    "id"             : { "type" : "string", "pattern" : "^([a-f]|[0-9]){7}$" },
+    "title"          : { "type" : "string", "maxLength": 160 },
+    "description"    : { "type" : "string" },
+    "classification" : { "type" : "string" },
+    "keywords"       : { "type" : "array", "items" : { "type" : "string", "maxLength" : 80 } },
+    "system"         : { "type" : "string", "maxLength" : 40 },
+    "severity"       : { "enum": [ "error", "warning", "convention-violation", "bad-smell", "minor-issue" ] },
+    "links"          : { "type" : "array", "items" : { "type" : "string", "format" : "uri" } },
+    "bug"            : { "type" : "object",
```

Comment:
Minor nitpick: I'd try to avoid aligning the `:` characters in this way. If a field in `properties` is added or changed, then you would potentially need to change the whitespacing for every line covered by `properties`, which leads to bigger diffs that are a little harder to parse.